modules/API/API.js

```diff
@@ -218,7 +218,18 @@ class API {
 
     /**
      * Sends remote control event.
-     * @param {object} event the event.
+     * @param {object} event the remote control event. The remote control event
```

Comment:
It would make more sense to move events descriptions to /service/remote/constants and describe each one there. Then just paste a reference or type here.
These links can be helpful:
http://usejsdoc.org/tags-typedef.html
http://usejsdoc.org/tags-enum.html
http://stackoverflow.com/questions/6460604/how-to-describe-object-arguments-in-jsdoc